google/pubsub/v1/pubsub_gapic.yaml

```diff
@@ -249,68 +249,71 @@ interfaces:
     field_name_patterns:
       topic: topic
     sample_code_init_fields:
-      - messages[0].data
+    - messages[0].data
```

Comment:
I vote for following PR. I'm doing some cleanup in preparation for mixin support.